src/main/java/LLMChat.java

```diff
@@ -0,0 +1,127 @@
+/*
+import okhttp3.*;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONException;
+import java.io.IOException;
+import java.nio.file.*;
+import java.util.Scanner;
+*/
```

Comment:
Although the imported libaries are block-commented out, I would suggest not to use wildcard when importing libraries for clarity